sbol/test/test_property.py

```diff
@@ -31,6 +31,26 @@ def test_removePropertyFromList(self):
         plasmid.removeRole()
         self.assertEqual(len(plasmid.roles), 1)
 
+    @unittest.skip("See #93")
```

Comment:
Can you use  @unittest.expectedFailure here? That would be preferable to skipping the test if it's possible.

Reply:
Now using expectedFailure